guide/the-scene-tree.md

```diff
@@ -30,6 +30,8 @@ value.
 ![](images/help-button.png =26x26) `Help`: Context sensitive help for the
 currently selected node.
 
+![](images/focus_button.png =26x26) `Focus`: Move viewpoint to focus on selected node.
```

Comment:
It was just an example of what I would write not exactly the sentence to write.
But I think that it is important to write at the results is that the object is at the center of the displayed image. You can use other words than scene if you think that they are better.